packages/pdf-generator/src/components/PDFDocument.js

```diff
@@ -52,19 +53,22 @@ const PDFDocument = ({
                 <View style={appliedStyles.smallSpacing}>
                     {page}
                 </View>
-                <View style={[ appliedStyles.flexRow, {
+                <View style={[{
```

Comment:
I'd change the styling to use `flex-direction: column` and wrapped the footer in `flexRow` that way someone using this would have easier way of passing multiple elements in the footer and not just one.

Reply:
I was intending for footer prop to be wrapped in view to choose custom bottom margin and such with less layout constraints, like this:

```
footer={
  <View style={{ marginBottom: 20 }}>
    <Text>{intl.formatMessage(messages.customReportFooter)}</Text>
  </View>
}
```
`

Reply:
Sure thing, I was just worried that folks using it won't know that wrapping it in `View` is the right way and wanted it to be easier for them, but that being said consumers have to use `Text` component as well. So good to go!